docs/guide/personalization/developer_guide/tracking_api.md

```diff
@@ -1,111 +1,173 @@
 # Tracking API
 
-If you are interested in importing historical user data, see [Importing historical user tracking data](importing_historical_user_tracking_data.md) as well. It speeds up enabling high quality recommendations.
+To provide recommendations, a tracking process needs to collect user behavior on the customer's site. 
+The most popular user events are:
 
-If you want to use a javascript library to add tracking into your website, see [Tracking with yct.js](tracking_with_yct.md).
+- Click - When a user opens a detail page.
+- Buy/Consume - When a user buys an item or consumes content
+- Rate - When a user likes, comments or rates an item
+- Login - When a user logs in on a website
+- Clickrecommended - When a user click a recommendation
 
-In order to provide recommendations a tracking process needs to collect user behavior on the customer's site. Most popular examples of such events are:
+For a complete list of events, see [Event types](https://doc.ibexa.co/projects/userguide/en/latest/personalization/event_types) in the user documentation. 
+Depending on the event type, some additional parameters, such as item price or user rating, must be provided.
 
-- "Click", if a user opens a detail page
-- "Buy" or "Consume", if a user buys an item or consumes in some other way (watches videos, reads articles, etc.)
-- "Rate" if a user likes, comments or rates an item
-- "Login" if a user logs in on a website and
-- "Clickrecommended" if a user decides to click on a recommendation
+Importing historical user data can help you reduce the delay in delivery of high quality recommendations. 
+For more information, see [Importing historical user tracking data](importing_historical_user_tracking_data.md).
 
-See user guide chapter [4. Event Types](https://doc.ezplatform.com/projects/userguide/en/master/personalization/event_types.md) for a brief introduction to different types of events. Depending on the event type, some additional parameters like item price or user rating must be provided.
+Apart from the tracking API, you can add tracking to the website by integrating a JavaScript library.
+For more information, see [Tracking with yct.js](tracking_with_yct.md).
 
-## Wording of products, articles and the generic term items
+## Definitions
 
-This documentation is primarily intended to be used for E-commerce and Publishers. It is hard to use generic wording in each domain as E-commerce mostly uses the term "product", whereas the Publisher domain widely uses the terms "content", "article", "images" or "videos". Because of this the generic term "item" will be used instead to cover all types, meaning "products", "content", "article", "images" or "videos". For further segmentation we use the term "itemtype" which in combination with the item ID itself defines a domain specific object.
+This document is intended for use in eCommerce and content publishing scenarios. 
```

Comment:
Aren't these the only two scenarios? ATM it sound like here were some others tat a documented elsewhere, so maybe like this:
```suggestion
You can use the tracking API both in eCommerce and content publishing scenarios. 
```